apostrophecms/apostrophe
# 3/edit relationship

Resolves https://apostrophecms.atlassian.net/jira/software/projects/A30U/boards/4?assignee=5ddbce7e47baa50cfd05eed2&selectedIssue=A30U-431

## How to test

- in a3-demo, edit the `article` schema by updating the `_product` field this way:
```js
_product: {
        type: 'relationship',
        max: 3,
        min: 2,
        withType: 'product',
        label: 'Linked product',
        fields: {
          add: {
            description: {
              type: 'string',
              label: 'Description'
            }
          }
        }
      }
```

- edit an article, add or edit a joined product
- in the slat list, there is a new icon (three dots). Click on it, and edit the relationship.
- a new modal will appear, add a value in "Description", save the article and go back to the article to check the "Description" value has been saved.
